vscode-powershell.build.ps1

```diff
@@ -132,14 +128,8 @@ task Package {
 
     Assert-Build (Test-Path ./dist/extension.js) "Extension must be built!"
 
-    # Packaging requires a copy of the modules folder, not a symbolic link. But
-    # we might have built in Debug configuration, not Release, and still want to
-    # package it. So delete the symlink and copy what we just built.
-    if ((Get-Item ./modules -ErrorAction SilentlyContinue).LinkType -eq "SymbolicLink") {
-        Write-Build DarkRed "PSES is a symbolic link, replacing with copy!"
-        Remove-BuildItem ./modules
-        Copy-Item -Recurse -Force "$(Split-Path (Get-EditorServicesPath))/module" ./modules
-    }
+    # With --follow-symlinks flag, VSCE can now package symbolic links directly,
```

Comment:
Instead of an orphaned comment that will be confusing in the future because of the lack of context, add an assertion like the one above that checks, PowerShell editor services has been built under modules.